nailgun/entities.py

```diff
@@ -6579,6 +6607,81 @@ def read(self, entity=None, attrs=None, ignore=None, params=None):
         return super(System, self).read(entity, attrs, ignore, params)
 
 
+class TemplateImport(Entity, EntityCreateMixin, EntityReadMixin):
+    """A representation of a Template Import entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._fields = {
+            'associate': entity_fields.StringField(),
+            'branch': entity_fields.StringField(),
+            'dirname': entity_fields.StringField(),
+            'filter': entity_fields.StringField(),
+            'force': entity_fields.StringField(),
+            'location_id': entity_fields.IntegerField(),
+            'lock': entity_fields.BooleanField(),
+            'negate': entity_fields.BooleanField(),
+            'organization_id': entity_fields.IntegerField(),
+            'prefix': entity_fields.StringField(),
+            'repo': entity_fields.StringField(),
+            'templates': entity_fields.ListField(),
```

Comment:
What's inside that list? Maybe it should be replaced with OneToManyField(SomeEntity)

Reply:
Yeah, Its output that import/export of templates returns ```[{'imported': yes/no, name:'some1'}, {{'imported': yes/no, name:'some2'}}]```(also many other attributes) for which we may need to create new entity class. Actually, I wanted to create that but I see nailgun doesn't have such things for any entity so just skipped my idea and stick with the list. 

Any thoughts ?

Reply:
Ok, not a blocker then. Pls just add some `# todo` comment so we won't forget